cmd/nerdctl/container_logs.go

```diff
@@ -0,0 +1,115 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/containerd/nerdctl/pkg/api/types"
+	"github.com/containerd/nerdctl/pkg/cmd/container"
+	"github.com/spf13/cobra"
+)
+
+func newLogsCommand() *cobra.Command {
```

Comment:
Hmm, I was following the pattern in #1869 as `container logs` is the canonical name of this command. @AkihiroSuda Do you know which pattern we should follow?

Reply:
ops, didn't notice that. I'm okay with either one. 

Currently it seems we only prefix `container`/`image` to file names if there're mutiple commands (e.g., `container_inspect.go/container_prune.go v.s. image_inspect.go/image_prune.go`; `run.go` instead of `container_run.go`).

But yeah adding a prefix can make those files closer :)

Reply:
My thoughts:

- Prefixing the files according to their canonical names may make them more organized.
- If we're grouping things according to their canonical names re. things under `pkg/cmd`, maybe we want to do so consistently in other places.

Just my 2 cents :)

Reply:
Yes, `containers_logs.go` is preferred as it corresponds to the canonical form `nerdctl containers logs`

Reply:
Wondering if I should do a one-shot PR for this. Maybe after #1900 is merged? /cc @Zheaoli

Reply:
Yes, @davidhsingyuchen, it's better to be merged after #1900